code/src/index.css

```diff
@@ -1,13 +1,273 @@
+* {
+  margin: 0 auto;
+  padding: 0;
+  box-sizing: border-box; 
+  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;
+}
+
 body {
+  background-color: #000;
+}
+
+/*MOVIE LIST/HOME PAGE*/
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.movie-card {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  position: relative;
+  margin: 0;
+  width: 100%;
+}
+
+
+.image {
+  object-fit: cover;
+  width: 100%;
+}
+
+.image-details {
+  position: absolute;
+  padding: 16px;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  background: rgba(0, 0, 0, 0.623);
+  transition: 0.5s ease;
+  opacity: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.released-date {
+  font-size: 16px;
+  padding-top: 12px;
+  font-weight: normal;
+  margin: 0;
+}
+
+.title {
+  margin: 0;
+  font-size: 32px;
+  font-weight: bold;
+}
+
+/*HOVER EFFECTS*/
+
+.image-details:hover {
+  opacity: 1;
+}
+
+/*MOVIE DETAILS PAGE*/
+.loading-spinner {
```

Comment:
I really like your way of naming your classes. It makes it easy to read and follow what you are attempting to style. Good job!